tests/unit/test_eel.py

```diff
@@ -0,0 +1,29 @@
+import eel
+import pytest
+from tests.utils import TEST_DATA_DIR
+
+# Directory for testing eel.__init__
+INIT_DIR = TEST_DATA_DIR / 'init_test'
+
+
+@pytest.mark.parametrize('js_code, expected_matches', [
+    ('eel.expose(w,"say_hello_js")', ['"say_hello_js"']),
+    ('eel.expose(function(e){console.log(e)},"show_log_alt")', ['"show_log_alt"']),
+    (' \t\nwindow.eel.expose((function show_log(e) {console.log(e)}), "show_log")\n', ['"show_log"']),
+    ((INIT_DIR / 'minified.js').read_text(), ['"say_hello_js"', '"show_log_alt"', '"show_log"']),
+    ((INIT_DIR / 'sample.html').read_text(), ['say_hello_js']),
+    ((INIT_DIR / 'App.tsx').read_text(), ["'say_hello_js'", "'show_log'"]),
+    ((INIT_DIR / 'hello.html').read_text(), ['say_hello_js', 'js_random']),
```

Comment:
Why are some of these quoted twice and others not? Are quotes sometimes part of the captured function name? That doesn't feel right 🤔

Reply:
Ah right - I see that these later get stripped of quotes. Is it possible to update the pyparsing grammar to exclude those for us, or is that too much of a pain? It would be nice to have that grammar only emit the names without us needing to do any further post-processing (but we can probably tolerate it as-is if the extra work is significant - or I can investigate it myself :))

Reply:
Yeah I hadn't look at this code in a little while so I had the same thought that the extra quotes were weird. I initially looked into handling the quotes in PEG and experimenting with quoted string, but that ended up not working as expected. I found that stripping the quotes after was just a bit cleaner

I have to wrap something up, but I can take a quick look tonight in the next hour or so. I think it *may* be straightforward to suppress the quotes as optional characters add quotes to the `excluded chars` for `pp.Word` match

Reply:
Got it!

```py
EXPOSED_JS_FUNCTIONS = pp.ZeroOrMore(
    pp.Suppress(
        pp.SkipTo(pp.Literal('eel.expose('))
        + pp.Literal('eel.expose(')
        + pp.Optional(
            pp.Or([pp.nestedExpr(), pp.Word(pp.printables, excludeChars=',')]) + pp.Literal(',')
        )
    )
    + pp.Suppress(pp.Regex(r'["\']?'))
    + pp.Word(pp.printables, excludeChars='"\')')
    + pp.Suppress(pp.Regex(r'["\']?\s*\)')),
)
```

Reply:
Nice!